tutorials/power_distribution/api.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+#include "libs/gpio/api.h"
+#include "libs/gpio/pin_defs.h"
+
+// Pins
+
+gpio_t DASHBOARD_POWER = PB0;
+gpio_t SERVICE_SECTION_POWER = PB1;
+gpio_t THROTTLE_POWER = PB2;
+
+// Macros
+
+/*
+Set power GPIO pin high.
+*/
+void power_device(gpio_t gpio_pin);
+
+/*
+Set dashboard power, service section power, and
+throttle power GPIO pins high.
+*/
+void power_all();
```

Comment:
Generally if there are no arguments, you can just put `void` to indicate that it is intentionally empty:

```c
void power_all(void);
```

Same below and in `.c` file.